argp-help.c

```diff
@@ -32,16 +32,45 @@
 #include <stdarg.h>
 #include <ctype.h>
 #include <limits.h>
+#ifndef _WIN32
 #include <features.h>
+#include <alloca.h>
+#endif
 #include <libintl.h>
 #include <locale.h>
-#include <alloca.h>
 #include <argp.h>
 #include "argp-fmtstream.h"
 #include "internal.h"
 #include <stdbool.h>
 #include <stdint.h>
 
+#ifdef _WIN32
```

Comment:
I think you could provide a standalone `argp-shim.h` file including stubs.

I may sync the code with the upstream later. It will be easier if these files did not change much.

Reply:
ok, done